root/main/404.js

```diff
@@ -26,14 +30,14 @@ const _404 = ({$c, ...props}) => (
         </p>}
       <p>
         {exp.l('Looking for help? Check out our {doc|documentation} or {faq|FAQ}.',
-           {doc: '/doc/MusicBrainz_Documentation', faq: '/doc/FAQ'})}
+               {doc: '/doc/MusicBrainz_Documentation', faq: '/doc/FAQ'})}
       </p>
       <p>
         {exp.l('Found a broken link on our site? Please {report|report a bug} and include any error message that is shown above.', {
           report: bugTrackerURL(
             'Nonexistent page: ' + $c.req.uri + '\n' +
-            'Referrer: ' + ($c.req.headers.referer || '')
-          )
+            'Referrer: ' + ($c.req.headers.Referer || ''),
```

Comment:
Our headers have uppercased Referer - this doesn't work otherwise.

Reply:
Hrm, that seems flaky, especially seeing as we reference `req.headers.host` in another file. I think I'd prefer if we explicitly lowercased them on the server during serialization.

Reply:
Would that involve         $headers{lc($name)} = $req->headers->header($name); in https://github.com/metabrainz/musicbrainz-server/blob/master/lib/MusicBrainz/Server.pm#L550 ?

Reply:
Yup, that should work. The header is not case-sensitive in the `->header` call.

Reply:
Ok, done (I think!)